contentcuration/contentcuration/tests/viewsets/test_clipboard.py

```diff
@@ -0,0 +1,282 @@
+from __future__ import absolute_import
+
+import uuid
+
+from django.core.management import call_command
+from django.core.urlresolvers import reverse
+from le_utils.constants import content_kinds
+
+from contentcuration import models
+from contentcuration.tests import testdata
+from contentcuration.tests.base import StudioAPITestCase
+from contentcuration.viewsets.sync.constants import CLIPBOARD
+from contentcuration.viewsets.sync.utils import generate_create_event
+from contentcuration.viewsets.sync.utils import generate_delete_event
+from contentcuration.viewsets.sync.utils import generate_update_event
+
+
+class SyncTestCase(StudioAPITestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.create_bucket()
```

Comment:
Is there a reason we're not just using `self.setUp` with persist_bucket set to False? https://github.com/learningequality/studio/blob/99e79d228452d5a4bb253eabd78919e369b7b3f0/contentcuration/contentcuration/tests/base.py#L101:107

Reply:
Yes - I needed to create the bucket during `setUpClass` in order for the files to be available for `setUpTestData` (which is invoked from `setUpClass`), this gets run before any `setUp` and so the test data setup would fail because the bucket and the files in the bucket did not exist.